rali/rali_pybind/amd/rali/plugin/tf.py

```diff
@@ -79,42 +74,83 @@ def __next__(self):
             self.loader.copyToTensorNCHW(self.out, self.multiplier, self.offset, self.reverse_channels, int(self.tensor_dtype))
         else:
             self.loader.copyToTensorNHWC(self.out, self.multiplier, self.offset, self.reverse_channels, int(self.tensor_dtype))
-
-        self.loader.getImageLabels(self.labels)
-        tf.reset_default_graph()
-        self.labels_tensor = tf.convert_to_tensor(self.labels,np.int32)
-
-        if self.tensor_dtype == types.FLOAT:
-            return tf.convert_to_tensor(self.out,np.float32), self.labels_tensor
-        elif self.tensor_dtype == types.FLOAT16:
-            return tf.convert_to_tensor(self.out,np.float16), self.labels_tensor
-
+        
+        if(self.loader._name == "TFRecordReaderDetection"):
+            sum = 0
```

Comment:
Codacy found an issue: [Redefining built-in 'sum'](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5976200)